Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -321,16 +247,139 @@ - (void)_stealFirstResponder;
 
 - (void)_showBugTitleCaptureAlert;
 {
-    UIAlertView *bugTitleCaptureAlert = [[UIAlertView alloc] initWithTitle:@"What Went Wrong?" message:@"Please briefly summarize the issue you just encountered. You’ll be asked for more details later." delegate:self cancelButtonTitle:@"Cancel" otherButtonTitles:@"Compose Report", nil];
-    bugTitleCaptureAlert.alertViewStyle = UIAlertViewStylePlainTextInput;
+    /*
+     iOS 8 often fails to transfer the keyboard from a focused text field to a UIAlertView's text field.
+     Transfer first responder to an invisble view when a debug screenshot is captured to make bug filing itself bug-free.
+     */
+    [self _stealFirstResponder];
+    
+    NSString * const title = NSLocalizedString(@"What Went Wrong?", nil);
+    NSString * const message = NSLocalizedString(@"Please briefly summarize the issue you just encountered. You’ll be asked for more details later.", nil);
+    NSString * const composeReportButtonTitle = NSLocalizedString(@"Compose Report", nil);
+    NSString * const cancelButtonTitle = NSLocalizedString(@"Cancel", nil);
     
-    UITextField *bugTitleTextField = [bugTitleCaptureAlert textFieldAtIndex:0];
-    bugTitleTextField.autocapitalizationType = UITextAutocapitalizationTypeSentences;
-    bugTitleTextField.autocorrectionType = UITextAutocorrectionTypeYes;
-    bugTitleTextField.spellCheckingType = UITextSpellCheckingTypeYes;
-    bugTitleTextField.returnKeyType = UIReturnKeyDone;
+#ifdef __IPHONE_8_0
```

Comment:
Ah, got it. The reason CI is failing then is you're using `_Nonnull` which is Xcode 7 only. You can  use `__nonnull` which will work in Xcode 6.3-7.0. You shouldn't need the `#ifdef __IPHONE_8_0`, since Aardvark's minimum Xcode version is Xcode 6.3, which has this API (sorry for the confusion!).

But you still need `if ([UIAlertController class])`, just in case you run on iOS 7.